getting-started/data-setup.rst

```diff
@@ -57,9 +57,11 @@ The Science Pipelines provides native integrations for many observatories, inclu
 
 First, clone `rc2_subset`_ using Git:
 
-.. code-block:: bash
+.. jinja:: default
+
+   .. code-block:: bash
 
-   git clone https://github.com/lsst-dm/rc2_subset
+     git clone -b {{ release_eups_tag }} https://github.com/lsst/rc2_subset
```

Comment:
Are you sure it's an EUPS tag here? I thought it would be a 24.0.0 not 24_0_0

Reply:
Indeed. Replaced it by release_git_ref